public/app/core/components/ColorScale/ColorScale.tsx

```diff
@@ -137,25 +137,31 @@ const getStyles = (theme: GrafanaTheme2, colors: string[]) => ({
   scaleGradient: css`
     background: linear-gradient(90deg, ${colors.join()});
     height: 10px;
+    pointer-events: none;
   `,
   legendValues: css`
     display: flex;
     justify-content: space-between;
+    pointer-events: none;
   `,
   hoverValue: css`
     position: absolute;
-    padding-top: 4px;
+    margin-top: -14px;
+    padding: 3px;
+    backdrop-filter: blur(6px);
```

Comment:
an alternative to this can be `background-color: rgba(0,0,0,0.5)` , and would have to become something like `background-color: rgba(255,255,255,0.5)`  for light theme